src/kubeconfig.ts

```diff
@@ -124,8 +124,7 @@ namespace KubeConfig {
         const ocOptions = Oc.getOptions({ flatten: "" });
 
         const execResult = await Oc.exec([ Oc.Commands.Config, Oc.Commands.View, ...ocOptions ],
-            { hideOutput: true /* Changing this breaks windows - See note about hideOutput in oc.exec */ }
-        );
+            { hideOutput: true /* Changing this breaks windows - See note about hideOutput in oc.exec */ });
```

Comment:
it should be:

```
const execResult = await Oc.exec(
    [ Oc.Commands.Config, Oc.Commands.View, ...ocOptions ],
    { hideOutput: true /* Changing this breaks windows - See note about hideOutput in oc.exec */ }
);
```

Reply:
I did the same way, but there was lint error.